frontend/src/components/worksheets/items/SchemaItem/SchemaItem.js

```diff
@@ -317,6 +314,7 @@ class SchemaItem extends React.Component<{
                                         return;
                                     }
                                     this.setState({ confirmingDeletion: true });
+                                    this.props.setDeleteItemCallback(this.deleteThisSchema);
```

Comment:
I think you need to set confirmingDeletion to false in the deleteThisSchema function, so the button will be re-enabled

Reply:
Thanks for pointing out the bug! But actually, I think we can delete the `confirmingDeletion` state variable since when we use dialog, all other buttons on the page will be automatically disabled.

Reply:
Ah makes sense, great!